classes/Validate.php

```diff
@@ -1269,7 +1269,7 @@ public static function isSiret($siret)
      */
     public static function isApe($ape)
     {
-        return (bool) preg_match('/^[0-9]{3,4}[a-zA-Z]{1}$/s', $ape);
+        return (bool) preg_match('/^[0-9]{1,2}?.[0-9]{1,2}[a-zA-Z]{1}$/s', $ape);
```

Comment:
Is the `.` part of your regex is correct?
With this regex, `12.12A` is valid but `12P12C`, `12345A`, `12A` also for instance.

Maybe this regex could be better:
```suggestion
        return (bool) preg_match('/^[0-9]{1,2}?\.?[0-9]{1,2}[a-zA-Z]{1}$/s', $ape);
```
(In test here: https://www.phpliveregex.com/p/K4M)

Reply:
You're right thanks !